src/udev/udev-builtin-net_id.c

```diff
@@ -31,6 +31,7 @@
  *                                         — USB port number chain
  *   v<slot>                               - VIO slot number (IBM PowerVM)
  *   a<vendor><model>i<instance>           — Platform bus ACPI instance id
+ *   i<addr>n<phys_port_name	           — Netdevsim bus address and port name
```

Comment:
Closing bracket is missing. Also, it seems that the `phys_port_name` part is conditional. It should be in square brackets.

Reply:
Will add the bracket. The phys_port_name part is not conditional. It is always present.